src/test/kotlin/org/rustSlowTests/cargo/runconfig/test/CargoTestRunnerTest.kt

```diff
@@ -48,6 +48,56 @@ class CargoTestRunnerTest : CargoTestRunnerTestBase() {
         """, sourceElement)
     }
 
+    fun `test doctests statuses`() {
+        buildProject {
+            toml("Cargo.toml", """
+                [package]
+                name = "sandbox"
+                version = "0.1.0"
+                authors = []
+            """)
+            dir("src") {
+                rust("lib.rs", """
+                    /// ```
+                    /// true;
+                    /// ```
+                    pub fn doctest_should_pass() {}
+                    /// ```
+                    /// assert_eq!(1, 2);
+                    /// ```
+                    pub fn doctest_should_fail() {}
+                    /// ```
+                    /// true;
+                    /// ```
+                    /// ```
+                    /// assert_eq!(1, 2);
+                    /// ```
+                    pub fn many_doctests() {}
+                    /// ```
+                    /// true;
+                    /// ```
+                    pub mod non_fn_doctest_should_pass {}
+                    /// ```
+                    /// assert_eq!(1, 2);
+                    /// ```
+                    pub mod non_fn_doctest_should_fail {}
```

Comment:
Let's add item(s) inside module to ensure we work correctly in such case as well